ext/js/dictionary/dictionary-importer.js

```diff
@@ -627,8 +625,6 @@ export class DictionaryImporter {
      */
     _convertTermBankEntryV3(entry, dictionary) {
         let [expression, reading, definitionTags, rules, score, glossary, sequence, termTags] = entry;
-        expression = this._normalizeTermOrReading(expression);
-        reading = this._normalizeTermOrReading(reading.length > 0 ? reading : expression);
```

Comment:
Same thing here. Is this intended? Or should it be:
```suggestion
        reading = reading.length > 0 ? reading : expression;
```